app/form_models/publishers/job_listing/application_form_form.rb

```diff
@@ -0,0 +1,77 @@
+class Publishers::JobListing::ApplicationFormForm < Publishers::JobListing::UploadBaseForm
+  CONTENT_TYPES_ALLOWED = %w[
+    application/pdf
+    application/msword
+    application/vnd.openxmlformats-officedocument.wordprocessingml.document
+  ].freeze
+
+  validate :application_form_presence
+  validates :application_email, presence: true
+  validate :other_application_email_presence
+  validate :other_application_email_valid
+  validates :how_to_apply, presence: true, if: -> { vacancy.how_to_apply.present? }
+
+  def self.fields
+    %i[how_to_apply application_email]
```

Comment:
We could remove `how_to_apply` because it's not needed for new listings and it's not editable for published legacy listings, and draft legacy listings aren't editable at all